README

```diff
@@ -0,0 +1,41 @@
+# Ngrok Agent Docker Image
```

Comment:
Style nit: `ngrok` is lowercase, even at the start of sentences.

`docker image` is lowercase as well since `Docker` evokes the company and `docker` evokes the open source tool derived from the moby project.

I think agent might be lowercase too, not sure about that one.